common/source/docs/common-lua-scripts.rst

```diff
@@ -289,6 +366,35 @@ The relay library proivdes access to controlling relay outputs.
 
 - :code:`toggle(relay_num)` - Toggles the requested relay on or off.
 
+
+Servo_Channels (SRV_Channels:)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :code:`find_channel(output function)` - Returns first servo output number -1 of an output assigned "output function" (See ``SERVOx_FUNCTION`` parameters ). False if none is assigned.
+
+Servo Output
+~~~~~~~~~~~~
+
+This is special case that is not generated via the binding methods.
+
+- :code:`servo.set_output (function number, PWM value)` -Sets servo outputs of type "function number" to PWM values between 1000 and 2000.
```

Comment:
The bounds are not strictly 1000 to 2000, it's whatever you want for a PWM value thats in the uint16_t number space, don't imply it has to be 1000 to 2000.

Reply:
fixed